### ISSUE TITLE
Unable to run openapi-generator due to low JDK version in docker base image

### ISSUE THREAD

**Issue by ityuhui** [kubernetes-client/gen#243]
The base image `maven:3.8.2-jdk-8-slim` can not run `openapi-generator`

https://github.com/kubernetes-client/gen/blob/f6b4147e445a6e26ac680e31da63fc42019e7735/openapi/openapi-generator/Dockerfile#L15

The error log is as follows:

```
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-release-plugin/2.5.3/maven-release-plugin-2.5.3.jar (53 kB at 1.8 MB/s)
Error:  Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.3.0:enforce (enforce-maven) on project openapi-generator-project: 
Error:  Rule 0: org.apache.maven.enforcer.rules.version.RequireJavaVersion failed with message:
Error:  Detected JDK version 1.8.0-302 (JAVA_HOME=/usr/local/openjdk-8/jre) is not in the allowed range [1.11.0,).
```

We should upgrade JDK 11 or later.

Which version/tag of docker image do you prefer @brendandburns ?
https://hub.docker.com/_/maven/tags

**Comment by ityuhui** [comment]
After this is fixed in the gen repo, kubernetes-client/c#188 and kubernetes-client/c#190 should be rolled back.

**Comment by brendandburns** [comment]
@ityuhui let's move to `[3.9.2-eclipse-temurin-11](https://hub.docker.com/layers/library/maven/3.9.2-eclipse-temurin-11/images/sha256-5e024ec8ab22c38e4994300094231067c49e1785eabea9e4fcf9082ca5b4f961?context=explore)`
